### ISSUE TITLE
ModuleNotFoundError when importing recipe_scrapers

### ISSUE THREAD

**Issue by bmease** [hhursev/recipe-scrapers#641]
**Pre-filing checks**

- [x] I have searched for open issues that report the same problem
- [x] I have checked that the bug affects the latest version of the library

**The version of Python you're using**
Python 3.10.6

**The operating system of your environment**
Linux

**The results you expect to see**
I'm getting a ModuleNotFoundError when importing recipe_scrapers. This seems to be from the recent commit 21811542f55b36de1b59d46cd9b6884a1160a84d which added some type checking.

**The results (including any Python error messages) that you are seeing**
```
>>> from recipe_scrapers import scrape_me
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/opt/venv/lib/python3.10/site-packages/recipe_scrapers/__init__.py", line 142, in <module>
    from .nihhealthyeating import NIHHealthyEating
  File "/opt/venv/lib/python3.10/site-packages/recipe_scrapers/nihhealthyeating.py", line 3, in <module>
    from attr import dataclass
ModuleNotFoundError: No module named 'attr'
```

- [ ] I'd like to try fixing this scraper myself
- [x] I'd like guidance to help me develop a fix
- [x] I'd prefer if the `recipe-scrapers` team try to fix this

**Comment by jayaddison** [comment]
Good catch, thanks @hay-kot.

We currently install our [development-time dependencies](https://github.com/hhursev/recipe-scrapers/blob/432663580c7a3e947fd7528345208bb983051b3e/requirements-dev.txt) before [running unit tests](https://github.com/hhursev/recipe-scrapers/blob/432663580c7a3e947fd7528345208bb983051b3e/.github/workflows/unittests.yaml#L39-L42), and it looks like `pytest` pulls in the `attrs` package, meaning that we didn't encounter this failure during continuous integration.

Possibly another vote for removing `pytest` as noted in #617 - and also more broadly a question of how to run Python unit tests where the package-under-test isn't able to import unit-test framework dependencies (this isn't something I know off the top of my head).

**Comment by jayaddison** [comment]
Oops, meant to thank both of you - thank you @bmease for reporting this.

**Comment by jayaddison** [comment]
This is now fixed in version 14.18.0, and published to PyPI.
